src/components/AdvancedSelect.vue

```diff
@@ -218,6 +218,7 @@
 <script type="text/javascript">
 import $ from 'jquery';
 import inView from 'in-view';
+import { clone } from 'ramda';
```

Comment:
`clone` is no longer used. Remove and uninstall

Reply:
Removed